migrate/livesync-for-s3.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Livesync S3 to Timescale Cloud
+excerpt: Synchronize data from S3 to Timescale Cloud service in real time
+products: [cloud]
+keywords: [migration, low-downtime, backup]
+tags: [recovery, logical backup, replication]
+---
+
+import PrereqCloud from "versionContent/_partials/_prereqs-cloud-only.mdx";
+import EarlyAccessNoRelease from "versionContent/_partials/_early_access.mdx";
+
+# Livesync from S3 to Timescale Cloud
+
+You use $LIVESYNC to synchronize all the data, or specific tables, from an S3 bucket to your
```

Comment:
`You use $LIVESYNC to synchronize tabular data`

Reply:
Done